packages/esm-patient-test-results-app/src/config-schema.ts

```diff
@@ -28,13 +28,25 @@ export const configSchema = {
       },
     ],
   },
-
   showPrintButton: {
     _type: Type.Boolean,
     _default: true,
     _description:
       'Whether or not to display the print button in the Test Results dashboard. When set to `true`, a print button is shown alongside the panel and tree view content switcher. When clicked, a modal pops up showing a datatable with the available test results. Once the user selects an appropriate date range, they can click on the print button in the modal to print the data',
   },
+  orders: {
+    // TODO: What is this and should it default to null (show all orders)?
+    careSettingUuid: {
+      _type: Type.UUID,
+      _description: 'Only orders corresponding to this care setting will be displayed.',
+      _default: '6f0c9a92-6f24-11e3-af88-005056821db0',
+    },
+    labOrderTypeUUID: {
```

Comment:
I think we're stuck with this convention:

```suggestion
    labOrderTypeUuid: {
```

Reply:
Good catch, thanks. Personally I like that convention. I think Copilot wrote it the other way and I didn't notice.